packages/libs/error-stack/README.md

```diff
@@ -97,6 +97,19 @@ This will most likely result in an error and print
 
 ![](https://github.com/hashintel/hash/blob/8ed55bd73045fba83a7ea2e199b31d5b829537b9/packages/libs/error-stack/assets/full.png?raw=true)
 
-Please see the [documentation] for a full description.
+## Usage
+
+Please see the [documentation].
+
+For more examples of `error-stack` in use, please check out the [examples](./examples) folder.
```

Comment:
I wonder if we want to refer to the examples folder by an absolute path as well, so the link is also working from external pages (like crates.io).

Reply:
@TimDiekmann Good shout.

```suggestion
For more examples of `error-stack` in use, please check out the [examples](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack/examples) folder.
```